setup.py

```diff
@@ -31,7 +31,7 @@
 
     install_requires=[
         'mkdocs',
-        'mkdocs-material>=6.2.1,<7.0',
+        'mkdocs-material==9.4.2',
```

Comment:
@jtherrmann do we want to be this strict with the material version, or constrain to the major version?

Reply:
No idea, we should discuss this further.